pkg/controller/clone-controller.go

```diff
@@ -0,0 +1,264 @@
+package controller
+
+import (
+	"fmt"
+	"time"
+	//"encoding/json"  temporary for debug
+	"github.com/golang/glog"
+	. "github.com/kubevirt/containerized-data-importer/pkg/common"
```

Comment:
This import path is defunct, please use `kubevirt.io/` throughout

Reply:
@zvikorn yeah, sorry about that. You'll need to rebase your branch, then move your cdi src tree from $GOPATH/src/github.com/containerized-data-importer to $GOPATH/src/kubevirt.io/containerized-data-importer